dev/io.openliberty.webcontainer.servlet.6.1.internal_fat/publish/servers/servlet61_ResponseSendRedirectTest/bootstrap.properties

```diff
@@ -0,0 +1,15 @@
+###############################################################################
+# Copyright (c) 2024 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+###############################################################################
+bootstrap.include=../testports.properties
+osgi.console=7777
+com.ibm.ws.logging.trace.specification=*=info:com.ibm.ws.webcontainer*=all:com.ibm.wsspi.webcontainer*=all:GenericBNF=all:io.openliberty.webcontainer*=all
```

Comment:
Same comment as previous PRs, do we need all of this trace? If not let's remove some of it.

Reply:
I will remove when I'm done with this item...

Reply:
Do you have an issue you're using to track this sort of change so we don't forget about it or do you plan to remove before merging?